.cirrus.yml

```diff
@@ -147,7 +147,8 @@ asan_task:
         `#--with-pdo-firebird`
         `#--with-pdo-dblib`
         --enable-werror
-        CFLAGS='-fsanitize=undefined,address -DZEND_TRACK_ARENA_ALLOC' LDFLAGS='-fsanitize=undefined,address'
+        CXXFLAGS='-fsanitize=undefined,address -DZEND_TRACK_ARENA_ALLOC -Wassign-enum'
```

Comment:
I think this was supposed to drop the `CXXFLAGS` instead of the `CFLAGS` :slightly_smiling_face:

Reply:
WTF, of course.